<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QYVideoWidget</class>
 <widget class="QWidget" name="QYVideoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color:rgba(26,26,26,255);
border-style:solid;
border-width:1px;
border-color:rgba(220,220,220,100);
}</string>
  </property>
  <widget class="QWidget" name="wd" native="true">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>632</width>
     <height>472</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
background-color:rgba(26,26,26,255);
border-width:0px;
}</string>
   </property>
   <property name="StopAspect" stdset="0">
    <number>1</number>
   </property>
   <widget class="QYVideoShow" name="wd_show" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>632</width>
      <height>472</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
background-color:rgba(26,26,26,255);
border-width:0px;
}</string>
    </property>
   </widget>
   <widget class="QYPixViewer" name="wd_pix" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>632</width>
      <height>472</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
border-width:0px;
}</string>
    </property>
    <property name="StopAspect" stdset="0">
     <number>1</number>
    </property>
   </widget>
   <widget class="QYDlgToolBar" name="wd_cross" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="wd_ctrl" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>364</y>
      <width>144</width>
      <height>108</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
color:rgba(200,207,217,255);
font-family:&quot;微软雅黑&quot;;
font-size:9px;
font-weight:bold;
background-color: rgba(64,64,64,255);
border-width: 0px;
border-style: solid;
border-radius:0px;
border-image:url(&quot;&quot;);
border-color:rgba(160,160,160,255);
}</string>
    </property>
    <property name="KeepAspect" stdset="0">
     <number>8</number>
    </property>
    <widget class="QDoubleSpinBox" name="spin_dr">
     <property name="geometry">
      <rect>
       <x>44</x>
       <y>14</y>
       <width>88</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
border-width:1px;
margin: 0px;
}
QDoubleSpinBox::down-arrow {
    image: url(&quot;:/QYRTSPWidget/2_0&quot;);
}
QDoubleSpinBox::up-arrow {
    image: url(&quot;:/QYRTSPWidget/2_1&quot;);
}
QDoubleSpinBox::up-button {
      subcontrol-origin: border;
      subcontrol-position: top right;
      border-width: 0px;

}
QDoubleSpinBox::up-button:pressed{
	background-color: rgba(200,200,200,255);
}
QDoubleSpinBox::down-button {
      subcontrol-origin: border;
      subcontrol-position: bottom right;
      border-width: 0px;

}
QDoubleSpinBox::down-button:pressed{
	background-color: rgba(200,200,200,255);
}
</string>
     </property>
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="minimum">
      <double>0.300000000000000</double>
     </property>
     <property name="maximum">
      <double>2.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.001000000000000</double>
     </property>
     <property name="value">
      <double>0.500000000000000</double>
     </property>
     <property name="StopAspect" stdset="0">
      <number>2</number>
     </property>
     <property name="KeepAspect" stdset="0">
      <number>4</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>14</y>
       <width>32</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
border-width:0px;
margin: 0px;
}</string>
     </property>
     <property name="text">
      <string>半径R</string>
     </property>
     <property name="KeepAspect" stdset="0">
      <number>2</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>46</y>
       <width>32</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
border-width:0px;
margin: 0px;
}</string>
     </property>
     <property name="text">
      <string>坐标X</string>
     </property>
     <property name="KeepAspect" stdset="0">
      <number>2</number>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="spin_dx">
     <property name="geometry">
      <rect>
       <x>44</x>
       <y>46</y>
       <width>88</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
border-width:1px;
margin: 0px;
}
QDoubleSpinBox::down-arrow {
    image: url(&quot;:/QYRTSPWidget/2_0&quot;);
}
QDoubleSpinBox::up-arrow {
    image: url(&quot;:/QYRTSPWidget/2_1&quot;);
}
QDoubleSpinBox::up-button {
      subcontrol-origin: border;
      subcontrol-position: top right;
      border-width: 0px;

}
QDoubleSpinBox::up-button:pressed{
	background-color: rgba(200,200,200,255);
}
QDoubleSpinBox::down-button {
      subcontrol-origin: border;
      subcontrol-position: bottom right;
      border-width: 0px;

}
QDoubleSpinBox::down-button:pressed{
	background-color: rgba(200,200,200,255);
}
</string>
     </property>
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="minimum">
      <double>0.300000000000000</double>
     </property>
     <property name="maximum">
      <double>0.600000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.001000000000000</double>
     </property>
     <property name="value">
      <double>0.500000000000000</double>
     </property>
     <property name="StopAspect" stdset="0">
      <number>2</number>
     </property>
     <property name="KeepAspect" stdset="0">
      <number>4</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>78</y>
       <width>32</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
border-width:0px;
}</string>
     </property>
     <property name="text">
      <string>坐标Y</string>
     </property>
     <property name="KeepAspect" stdset="0">
      <number>2</number>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="spin_dy">
     <property name="geometry">
      <rect>
       <x>44</x>
       <y>78</y>
       <width>88</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
border-width:1px;
}
QDoubleSpinBox::down-arrow {
    image: url(&quot;:/QYRTSPWidget/2_0&quot;);
}
QDoubleSpinBox::up-arrow {
    image: url(&quot;:/QYRTSPWidget/2_1&quot;);
}
QDoubleSpinBox::up-button {
      subcontrol-origin: border;
      subcontrol-position: top right;
      border-width: 0px;

}
QDoubleSpinBox::up-button:pressed{
	background-color: rgba(200,200,200,255);
}
QDoubleSpinBox::down-button {
      subcontrol-origin: border;
      subcontrol-position: bottom right;
      border-width: 0px;

}
QDoubleSpinBox::down-button:pressed{
	background-color: rgba(200,200,200,255);
}
</string>
     </property>
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="minimum">
      <double>0.300000000000000</double>
     </property>
     <property name="maximum">
      <double>0.600000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.001000000000000</double>
     </property>
     <property name="value">
      <double>0.500000000000000</double>
     </property>
     <property name="StopAspect" stdset="0">
      <number>2</number>
     </property>
     <property name="KeepAspect" stdset="0">
      <number>4</number>
     </property>
    </widget>
    <widget class="QPushButton" name="butCtrlShow">
     <property name="geometry">
      <rect>
       <x>134</x>
       <y>0</y>
       <width>10</width>
       <height>10</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
border-width:0px;
margin: 0px;
background-color:rgba(200,200,200,128);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="KeepAspect" stdset="0">
      <number>9</number>
     </property>
    </widget>
   </widget>
   <zorder>wd_cross</zorder>
   <zorder>wd_show</zorder>
   <zorder>wd_pix</zorder>
   <zorder>wd_ctrl</zorder>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QYPixViewer</class>
   <extends>QWidget</extends>
   <header>qypixviewer.h</header>
  </customwidget>
  <customwidget>
   <class>QYDlgToolBar</class>
   <extends>QWidget</extends>
   <header>qydlgtoolbar.h</header>
  </customwidget>
  <customwidget>
   <class>QYVideoShow</class>
   <extends>QWidget</extends>
   <header>QYVideoShow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
